### PR TITLE
208 cloud proxy doc for electron

### PR DESCRIPTION
**Pull request by hussain-khalid**

### CODE REVIEW

File: shared/video-sdk/develop/cloud-proxy/project-setup/electron.mdx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<PlatformWrapper platform="electron">
+
```
Comment (Kishan-Dhakan) [inline shared/video-sdk/develop/cloud-proxy/project-setup/electron.mdx:2]: In docs/shared/video-sdk/develop/_cloud-proxy.mdx (not changed as a part of this PR but still can be fixed in this) `[<Vpl k="CONSOLE" />]` is not rendering and needs to be changed to `[<Vg k="CONSOLE" />]`

File: shared/video-sdk/develop/cloud-proxy/project-implementation/electron.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<PlatformWrapper platform="electron">
+
+1.  **Import the required modules**
+
+    In `preload.js`, add the following before `createAgoraRtcEngine`:
```
Comment (Kishan-Dhakan) [inline shared/video-sdk/develop/cloud-proxy/project-implementation/electron.mdx:5]: Missing comma in `createAgoraRtcEngine`, i.e., it should be ``createAgoraRtcEngine,`.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
MInor changes requested, LGTM otherwise

**Review by billy-the-fish — APPROVED** [review]

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/electron.mdx [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+### Implement the user interface
+
+To implement the user interface, replace the contents of `index.html` with the following:
+<ProductWrapper product="video-calling">
+    ```html
+    <!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <title>Get started with <Vpd k="PRODUCT" /></title>
```
[the remark this answers is not in the document]
Comment (hussain-khalid) [reply shared/video-sdk/get-started/get-started-sdk/project-implementation/electron.mdx:12]: i have removed these variables already.

**Pull request merged**